fetch/client/src/main/scala/endpoints4s/fetch/EndpointsSettings.scala

```diff
@@ -0,0 +1,37 @@
+package endpoints4s.fetch
+
+import endpoints4s.Hashing
+
+import scala.annotation.nowarn
+
+final class EndpointsSettings(val host: Option[String]) extends Serializable {
```

Comment:
```suggestion
final class EndpointsSettings private (val host: Option[String]) extends Serializable {
```

The expected way to build an `EndpointsSettings` with a host would be:

~~~ scala
EndpointsSettings().withHost(Some("..."))
~~~